rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -80,5 +80,8 @@ build_rootfs() {
 		${ROOTFS_DIR}
 
 	chroot $ROOTFS_DIR ln -s /lib/systemd/systemd /usr/lib/systemd/systemd
-}
 
+	# Reduce image size and memory footprint
+	# removing not needed files and directories.
+	chroot $ROOTFS_DIR rm -rf /usr/share/{bash-completion,bug,doc,info,lintian,locale,man,menu,misc,pixmaps,terminfo,zoneinfo,zsh}
```

Comment:
This looks fine, except for `doc`. For that directory we probably want to:

- Keep all copyright files, but compress them.
- Delete everything else.

Here's what Ubuntu Core does:

- https://git.launchpad.net/livecd-rootfs/tree/live-build/ubuntu-core/hooks/10-remove-documentation.binary

It might be worth checking all the hooks in fact to see if we can reduce the size even further:

- https://git.launchpad.net/livecd-rootfs/tree/live-build/ubuntu-core/hooks